x-pack/plugins/synthetics/public/legacy_uptime/app/uptime_overview_fetcher.ts

```diff
@@ -26,6 +28,7 @@ async function fetchUptimeOverviewData({
     dateStart: start,
     dateEnd: end,
     bucketSize: intervalString,
+    timeZone: timeZone ?? moment.tz.guess(),
```

Comment:
```suggestion
    timeZone: timeZone,
```

if I understand the code correctly it should fall back already to this